src/helpers/http.js

```diff
@@ -18,6 +19,18 @@ const URL_BLACKLIST = {
     '169.254.169.254': 'startsWith', // EC2 document ip. Have better filtering in the future
 };
 
+
+// Brotli decompression exists since Node v10
+const ENCODING_FUNCTIONS = {
+    br: zlib.brotliDecompressSync,
+    brotli: zlib.brotliDecompressSync,
+    gzip: zlib.gunzipSync,
+    deflate: zlib.deflateSync,
+};
+
+// Whether to try and decode HTTP response
+const DECODE_HTTP = (process.env.EPSAGON_DECODE_HTTP || 'TRUE').toUpperCase() === 'TRUE';
```

Comment:
it would be probably good to have this as config option as well.

Reply:
👍